doc/source/contributor/environments/ci-aio.rst

```diff
@@ -11,8 +11,29 @@ Prerequisites
 * a CentOS Stream 8 or Ubuntu Focal 20.04 host
 * access to the Test Pulp server on SMS lab
 
```

Comment:
Needs a sentence here to explain that there are two methods.

Reply:
I've added it a bit higher, above the prerequisites section